logic/staff/create.go

```
@@ -14,194 +12,119 @@ import (
-	// 创建账号
-	if err := tx.Save(&account).Error; err != nil {
-		tx.Rollback()
+	if err := l.Db.Create(&l.Staff).Error; err != nil {
 		return errors.New("创建账号失败")
 	}
 
+	return nil
+}
+
+// 发送消息
+func (l *StaffCreateLogic) sendMessage() error {
```

golangci-lint (1.64.8)

func `(*StaffCreateLogic).sendMessage` is unused

(unused)